indra/newview/scripts/lua/require/LLChat.lua

```diff
@@ -14,4 +18,25 @@ function LLChat.sendShout(msg)
     leap.send('LLChatBar', {op='sendChat', type='shout', message=msg})
 end
 
+-- 0 is public nearby channel, other channels are used to communicate with LSL scripts
+function LLChat.sendChannel(msg, channel)
+    leap.send('LLChatBar', {op='sendChat', channel=channel, message=msg})
+end
```

Comment:
As far as I'm concerned, you could just add a `channel` parameter to `sendNearby()` and add `channel=channel` to the event table at line 10. If the caller passes only the `msg` parameter, `channel` will be `nil`, and setting a Lua table key to `nil` deletes it from the table -- making it exactly like the present `sendNearby()` event table.

I like the comment, though. Definitely keep that.

Reply:
Sure, it makes sense.